include/zephyr/net/net_event.h

```diff
@@ -132,13 +132,13 @@ enum net_event_ipv6_cmd {
 	(_NET_EVENT_IPV6_BASE | NET_EVENT_IPV6_CMD_NBR_DEL)
```

Comment:
Minor nit in the commit subject, no need to mention net so many times. Please change it to
`net: mgmt: Fix for wrong IPv6 DHCP event code`